BackofficeBundle/Resources/public/coffee/dataTable/DataTableView.coffee

```diff
@@ -0,0 +1,298 @@
+OpenOrchestraDataTableChannel = new (Backbone.Wreqr.EventAggregator)
```

Comment:
This channel could have been added to the OpenOrchestra js namespace (see groupTreeLib.coffee)